data/sql/updates/pending_db_world/rev_1739897580268046640.sql

```diff
@@ -0,0 +1,110 @@
+DROP TABLE IF EXISTS `antidos_opcode_policies`;
+CREATE TABLE `antidos_opcode_policies` (
+  `Opcode` smallint unsigned NOT NULL,
+  `Policy` tinyint unsigned NOT NULL,
+  `MaxAllowedCount` smallint unsigned NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
```

Comment:
```suggestion
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Reply:
Should opcode be able to have several entries here?
If not I suggest setting it as primary key. If you can have more than one policy I would suggest PRI Opcode and Policy.

Alternative make policy a bitmask to only have one opcode entry

Reply:
Shouldn't be several entries no, it would make no sense (and not properly handled) to have multiple policies either. I've added the primary key to Opcode.